api/src/main/java/javax/mvc/binding/BindingResult.java

```diff
@@ -99,31 +99,31 @@
     BindingError getBindingError(String param);
 
     /**
-     * Returns an immutable set of all constraint violations detected.
+     * Returns an immutable set of all validation errors detected.
      *
-     * @return All constraint violations detected
+     * @return All validation errors detected
      */
-    Set<ConstraintViolation<?>> getAllViolations();
+    Set<ValidationError> getAllValidationErrors();
```

Comment:
I'd like to call it `getValidationErrors` (without the implicit "all" prefix)
so it gonna be:
```
getValidationErrors() -> all
getValidationErrors(String param) -> all errors for specified param
getValidationError(String param) -> single error for specified param
```
what do you think @chkal?

note: this PR sounds great, thanks! +1 for merging it